src/content/advanced/gitops/index.md

```diff
@@ -125,367 +123,193 @@ flux-multi-tenancy:
-
-File structure
+We can see if the change was applied using `kubectl` (names will difer):
 
-```nohighlight
-.
-├── 03-cluster-aws.yaml     # contains HelmRelease CR
-├── 03-cluster-aws          # Helm release directory
-│   ├── Chart.yaml          # Helm chart definition
-│   ├── templates
-│   │   └── resources.yaml  # contains Giant Swarm cluster CRs
-│   └── values.yaml         # Helm chart values
+```nohiglight
+kubectl get gitrepositories -n default 
+NAME        URL                                       READY   STATUS                                                            AGE
+flux-demo   https://github.com/giantswarm/flux-demo   True    Fetched revision: main/74f8d19cc2ac9bee6f45660236344a054c63b71f   27s
 ```
 
-```nohighlight
-mkdir -p 03-cluster-aws/templates
-```
+or Flux CLI - `flux`:
 
 ```nohighlight
-kubectl gs template cluster --provider aws --name demo0 --organization flux-demo --description flux-demo > 03-cluster-aws/templates/cluster-resources.yaml
+ flux get sources git -n default 
+NAME            READY   MESSAGE                                                         REVISION                                        SUSPENDED
+flux-demo       True    Fetched revision: main/74f8d19cc2ac9bee6f45660236344a054c63b71f main/74f8d19cc2ac9bee6f45660236344a054c63b71f   False
 ```
 
-> Note: There are some parameterization and hooks already added in the demo repository. You can copy from there if you don't want to create your own.
-> Otherwise please:
->
-> - parameterize common variables and put them in `values.yaml`: cluster ID, organization name, namespace, control plane ID, etc.
-> - add hook annotations to `Cluster` CR: `helm.sh/hook: pre-install` and `helm.sh/hook-weight: "-1"`
+Next, we configure the keys that will be used in the management cluster flux to decypher secrets so they can be safely stored in the repo.
 
-The cluster will still need a NodePool to reach full functionality. To learn more, visit [NodePool documentation]({{< relref "/advanced/node-pools/index.md" >}}). We will be creating one using `kubectl gs template` command.
+In this example we are using the simplest for of `sops` key management, we are creating a master key for all the kustomizations in this management cluster.
 
-```nohighlight
-kubectl gs template nodepool \
-        --provider aws \
-        --cluster-name demo0 \
-        --description demo0 \
-        --organization flux-demo \
-        --availability-zones eu-west-1a > 03-cluster-aws/templates/nodepool-resources.yaml
-```
-
-Please note that `kubectl gs template nodepool` requires you to specify availability zones by name and they may vary depending on your region.
+The key was already created for us in the `kubectl gs gitops add management-cluster` command so we only need to apply the secret in the cluster and safely store the private gpg key so we can use it to encrypt files before pushing them to the repo.
```

Comment:
```suggestion
The key was already created for us in the `kubectl gs gitops add management-cluster` command so we only need to apply the secret in the cluster and safely store the private GPG key so we can use it to encrypt files before pushing them to the repo.
```
Instead of put the command we could add an anchor to the step